samples/excel/85-preview-apis/direct-precedents.yaml

```
@@ -0,0 +1,154 @@
+order: 6
+id: excel-direct-precedents
+name: Direct precedents
+description: 'This sample shows how to highlight precedent cells which directly affect the value of the currently selected cell, using the direct precedents API.'
```

Consider rephrasing this to explicitly mention formulas.
```suggestion
description: 'This sample shows how to find and highlight the precedents of the currently selected cell. Precedents  are cells referenced by the formula in a cell.'
```